tm2/pkg/bft/config/config.go

```diff
@@ -115,9 +115,6 @@ func (cfg *Config) EnsureDirs() {
 // ValidateBasic performs basic validation (checking param bounds, etc.) and
 // returns an error if any check fails.
 func (cfg *Config) ValidateBasic() error {
-	if err := cfg.BaseConfig.ValidateBasic(); err != nil {
-		return err
-	}
```

Comment:
sure?

Reply:
This validation only validates the logger configuration in the base config, which was dropped in this PR:

https://github.com/gnolang/gno/blob/6112ffd31f68131fdf07bc61408ddd32e5d6f38b/tm2/pkg/bft/config/config.go#L297-L306

Reply:
I think we should retain the ValidateBasic, even if it currently serves no purpose.

Looking for additional opinions.

Reply:
This is a bit out of the scope of this PR, but I've added actual basic base config verification we never had before:

[89c7b62](https://github.com/gnolang/gno/pull/1302/commits/89c7b6253213b111a577a44803fe556c21dd4ce3)


I'll open up a series of PRs to tackle the config, but for now this should be a good start